internal/resolvers/youtube/helpers.go

```diff
@@ -17,3 +30,17 @@ func getYoutubeVideoIDFromURL(url *url.URL) string {
 func getYoutubeVideoIDFromURL2(url *url.URL) string {
 	return path.Base(url.Path)
 }
+
+func getYoutubeChannelIdFromURL(url *url.URL) channelID {
```

Comment:
```suggestion
func getYoutubeChannelIDFromURL(url *url.URL) channelID {
```

Will need to update any references to it too

Reply:
fixed it in a commit instead :tf: